subprojects/ide-native/src/integTest/groovy/org/gradle/ide/xcode/XcodeMultipleCppProjectIntegrationTest.groovy

```diff
@@ -17,17 +17,146 @@
 package org.gradle.ide.xcode
 
 import org.gradle.ide.xcode.fixtures.AbstractXcodeIntegrationSpec
+import org.gradle.integtests.fixtures.executer.GradleContextualExecuter
+import org.gradle.nativeplatform.fixtures.app.CppAppWithLibraries
 import org.gradle.nativeplatform.fixtures.app.CppAppWithLibrary
+import org.gradle.util.Requires
+import org.gradle.util.TestPrecondition
+import spock.lang.IgnoreIf
 
 import static org.gradle.ide.xcode.internal.XcodeUtils.toSpaceSeparatedList
 
+@Requires(TestPrecondition.XCODE)
+@IgnoreIf({GradleContextualExecuter.embedded})
```

Comment:
We should never ignore tests for embedded execution, otherwise the test won't run during any dev builds, or from IDEA, or during the early stages of the CI pipeline. We should instead fix the tests to work in this mode.

Reply:
Very good point. Thanks to Sterling I changed to use `executer.requireGradleDistribution()` to make Xcode happy.